api/serializers.py

```diff
@@ -134,3 +134,22 @@ class Meta:
         model = AssetType
         fields = ("id", "asset_type", "asset_sub_category",
                   "created_at", "last_modified")
+
+
+class AssetModelNumberSerializer(serializers.ModelSerializer):
```

Comment:
I think we can use this serializer for all actions but override `create` to get the id from the `validated_data`.. we'd still need `get_make_label ` though

Reply:
@egichuri once I define `make_label` as a `SerializerMethodField` it then is not included in the `validated_data`